ci-operator/step-registry/openshift/image-based/upgrade/seed/create/openshift-image-based-upgrade-seed-create-commands.sh

```diff
@@ -102,6 +102,11 @@ cd ${remote_workdir}/ib-orchestrate-vm
 # Create the seed vm
 make seed
 
+if [[ "${CREATE_CLUSTER_ONLY}" == "true" ]]; then
```

Comment:
Naive question, why not split this step up in two steps so that they can be added individually instead of having a flag?

Reply:
Primarily because the logic of the steps is 95% the same, from a maintenance standpoint, it's just easier to have the flag